tests/integration/models/release.spec.js

```diff
@@ -354,6 +396,23 @@ describe('Release Model', function () {
 						});
 					});
 			});
+
+			it('should get the requested release by raw_version', function () {
+				return balena.models.release
+					.get({
+						rawVersion: ctx.currentRelease.raw_version,
+						application: ctx.application.id,
+					})
```

Comment:
Might be nice to start to use async await as we do above (absolutely not a blocker, just a preference)